ethereum_proxy/src/GoRpc/contracts/battlegroups.go

```diff
@@ -176,10 +169,10 @@ func (_BattleGroups *BattleGroupsTransactorRaw) Transact(opts *bind.TransactOpts
 	return _BattleGroups.Contract.contract.Transact(opts, method, params...)
 }
 
-// MAX_CARDS_PER_GROUP is a free data retrieval call binding the contract method 0xbfeea708.
+// MAXCARDSPERGROUP is a free data retrieval call binding the contract method 0xbfeea708.
```

Comment:
What happened to the `_`?

Reply:
idk, this is autogenerated code

Reply:
Just wanted to make sure it wasn't an outdated autogen, possibly from a moment when there were temporarily no '_''s.

Reply:
Nah it's not outdated, it runs everytime docker composes, the tool is just finicky, I think this occurred when we changed to the int literal instead of a const creation